filebeat/module/postgresql/csv/_meta/fields.yml

```diff
@@ -0,0 +1,71 @@
+- name: log
```

Comment:
This is going to generate duplicated field groups, this name should be unique in the module.
```suggestion
- name: csv
```